README.md

```diff
@@ -211,7 +209,6 @@ Table of Contents
 - [**Emerald Launcher**](https://github.com/HenriDellal/emerald) <sup>**[[F-Droid](https://f-droid.org/app/ru.henridellal.emerald)]**</sup>
 - [**Rootless Pixel Launcher**](https://github.com/amirzaidi/Launcher3) <sup>**[[F-Droid](https://f-droid.org/app/amirz.rootless.nexuslauncher)]**</sup>
 - [**BlissLauncher**](https://gitlab.e.foundation/e/apps/BlissLauncher) <sup>**[[F-Droid](https://f-droid.org/app/foundation.e.blisslauncher)]**</sup>
-- [**Posidon Launcher**](https://github.com/leoxshn/posidonLauncher) <sup>**[[F-Droid](https://f-droid.org/app/posidon.launcher)]**</sup>
```

Comment:
What a pity, it was really promising. While the website is still up and the last release wasn't too long ago, I couldn't find any infos about what happened to the project unfortunately. @leoxshn seems still active on GH though.

Reply:
Yeah! It was a good launcher though. Will see if anyone forks and maintains it